### PR TITLE
fix(580): add annotations to pipeline model during sync

### PR DESCRIPTION
**Pull request by minzcmu**
add annotations to pipeline model during pipeline.sync()

Related to https://github.com/screwdriver-cd/screwdriver/issues/580

Blocked by https://github.com/screwdriver-cd/data-schema/pull/147

### CODE REVIEW

File: test/lib/pipeline.test.js [hunk 1]
```diff
@@ -268,6 +268,17 @@ describe('Pipeline Model', () => {
             });
         });
 
+        it('store annotations to pipeline', () => {
```
Comment (r3rastogi) [inline test/lib/pipeline.test.js]: 'store' --> 'stores'

**Review by r3rastogi — APPROVED** [review]
Minor comment, looks good

**Review by ian-fox — APPROVED** [review]

File: test/lib/pipeline.test.js [hunk 2]
```diff
@@ -268,6 +268,17 @@ describe('Pipeline Model', () => {
...
         });
 
+        it('stores annotations to pipeline', () => {
+            jobs = [];
+            jobFactoryMock.list.resolves(jobs);
+            jobFactoryMock.create.withArgs(publishMock).resolves(publishMock);
+            jobFactoryMock.create.withArgs(mainMock).resolves(mainMock);
+
+            return pipeline.sync().then(() => {
+                assert.deepEqual(pipeline.annotations, {});
```
Comment (d2lam) [inline test/lib/pipeline.test.js]: Should probably test with something that has annotations specified

**Review by d2lam — CHANGES_REQUESTED** [review]

File: test/lib/pipeline.test.js (hunk 2 above)
Comment (minzcmu) [reply test/lib/pipeline.test.js]: fixed

**Review by d2lam — APPROVED** [review]

**Pull request merged**